repos/system_upgrade/el7toel8/actors/scangrubdevpartitionlayout/libraries/scan_layout.py

```diff
@@ -64,7 +64,6 @@ def get_partition_layout(device):
         if not line.startswith('Device'):
             continue
 
-        part_all_attrs = split_on_space_segments(line)
         break
```

Comment:
@mscherer  actually, checking the code, it seems this whole loop can be removed, as the only purpose is to get `part_all_attrs` variable.